cli/cmd/generate_gcp.go

```diff
@@ -0,0 +1,787 @@
+package cmd
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+
+	"github.com/lacework/go-sdk/internal/file"
+	"github.com/lacework/go-sdk/lwgenerate/gcp"
+)
+
+var (
+	// Define question text here to be reused in testing
+	QuestionGcpEnableConfig            = "Enable Config Integration?"
```

Comment:
Based on our Slack message https://lacework.slack.com/archives/CUTNH5QKY/p1646692456149189

@Karmali how does this sound? 🤔 `Enable compliance integration?` - Does that sound accurate?

cc @marcosgm 
```suggestion
	QuestionGcpEnableConfig            = "Enable compliance integration?"
```

Reply:
I'm with @marcosgm on this. Considering we have this integration names config in the UI & docs and the module itself is also called config, I'd prefer to keep it as is.

Reply:
I think @marcosgm said to keep it as `configuration` so:
```
Enable configuration integration?
```

Reply:
@marcosgm just want to double-check. Are we okay to use config here? This will match what we have in AWS and Azure and in all of our public-facing docs for this integration type.

Reply:
The documentation has "Config" instead of "Compliance" in the integration doc articles. That terminology is in a lot of places in the external documentation actually. @marcosgm can you raise a JIRA ticket with the doc_impact field as doc-review so that the doc team can evaluate/modify those instances.

@marcosgm Are we thinking that the term "Config" is bad even when we are talking about GCP and Azure? I realize it is problematic with AWS because there is an AWS Config term.

Reply:
@marcosgm Are we thinking that the term "Config" is bad even when we are talking about GCP and Azure? I realize it is problematic with AWS because there is an AWS Config term.  But I'm not sure there is such a concern with GCP/Azure. But then we run into inconsistent usage of "Config" vs "Compliance" for different CSPs...